recipes/libunwind/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX
+make
+# make check
```

Comment:
What sort of issues were encountered here?

Reply:
It looked like a single test failed (`run-core-dump-unwind`) with all others passing. I couldn't find anything relevant on the libunwind email list pointing to a specific issue.

Reply:
Do you feel it necessary that all of the tests pass? The only thing I could find was this thread that had no resolution:

https://lists.nongnu.org/archive/html/libunwind-devel/2015-01/msg00010.html

Reply:
If it were me, I'd patch out the broken test and link to that thread in the recipe comments. The motivation being I'd want to be sure we had tested something this low level to the fullest extent possible.

Reply:
It's not entirely clear to me where to patch out the test. This is mostly due to my unfamiliarity with autotools and all of the generated files. There are `Makefile.in` and `Makefile.am` files both at the top level of the source distribution, but also in the `tests` directory. I'm not 100% sure where the test is actually being run from.

Reply:
I think you might be able to get away with just deleting the file `tests/run-coredump-unwind`. At least that would be an easy first thing to try.

Reply:
Ok, I commented out the logic in `tests/run-coredump-unwind` as a patch and that seems to have worked . All other tests pass and I've added a comment above the `patches` entry in the `meta.yaml`.

Reply:
SGTM. Might want to bump that thread. Though it is pretty old.